classquiz/db/models.py

```diff
@@ -275,7 +254,7 @@ class AnswerData(BaseModel):
 
 class AnswerDataList(BaseModel):
     # Just a method to make a top-level list
-    __root__: list[AnswerData]
+    root: list[AnswerData]
```

Comment:
Does that work like that in new pydantic?

Reply:
Yes, that's right!

Reply:
https://docs.pydantic.dev/2.7/concepts/models/#rootmodel-and-custom-root-types